bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/api/AgsAnalysis.scala

```diff
@@ -210,4 +214,20 @@ object AgsAnalysis {
       analysisOpt.getOrElse(NoMagnitudeForBand(guideProbe, guideStar))
     }
   }
-}
\ No newline at end of file
+}
+
+sealed trait GuideInFOV
+case object InsideFOV extends GuideInFOV
+case object OutsideFOV extends GuideInFOV
```

Comment:
You might consider putting these on the companion so you could say `GuideInFOV.Inside` and `GuideInFOV.Outside`. Extremely minor but I'm liking that style better these days.